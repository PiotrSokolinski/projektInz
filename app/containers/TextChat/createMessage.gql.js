import gql from 'graphql-tag'

export default gql`
  mutation createMessage($id: ID!, $text: String!) {
    createMessage(id: $id, text: $text) {
      id
    }
  }
`
