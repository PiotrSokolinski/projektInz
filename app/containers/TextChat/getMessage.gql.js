import gql from 'graphql-tag'

export default gql`
  subscription newMessageCreated {
    newMessageCreated {
      id
      text
      sender {
        id
        firstName
        lastName
        avatarUrl
        color
      }
    }
  }
`
