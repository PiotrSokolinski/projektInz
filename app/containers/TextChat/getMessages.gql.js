import gql from 'graphql-tag'

export default gql`
  query getMessages($id: ID!, $skip: Int!, $take: Int!) {
    getMessages(id: $id, skip: $skip, take: $take) {
      id
      text
      createdAt
      sender {
        id
        firstName
        lastName
        avatarUrl
        color
      }
    }
  }
`
