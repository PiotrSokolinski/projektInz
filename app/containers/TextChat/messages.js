/*
 * TextChat Messages
 *
 * This contains all the text for the TextChat container.
 */

import { defineMessages } from 'react-intl'

export const scope = 'app.containers.TextChat'

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'Chat with {name}',
  },
  textAreaPlaceholder: {
    id: `${scope}.textAreaPlaceholder`,
    defaultMessage: 'Start typing...',
  },
  send: {
    id: `${scope}.send`,
    defaultMessage: 'Send',
  },
  you: {
    id: `${scope}.you`,
    defaultMessage: 'You',
  },
  noMoreMessages: {
    id: `${scope}.noMoreMessages`,
    defaultMessage: 'No more messages',
  },
})
