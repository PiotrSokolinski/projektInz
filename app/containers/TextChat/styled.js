import styled from 'styled-components'
import Textarea from 'react-textarea-autosize'
import { Attachment } from 'styled-icons/icomoon/Attachment'
import Button from 'components/Button'
import { Colors } from 'themes'

export const AttachmentIcon = styled(Attachment)`
  color: ${Colors.ecstasy};
  margin: 0 15px;
  cursor: pointer;
`

export const SendButton = styled(Button)`
  width: 100px;
  height: 30px;
`

export const Container = styled.div`
  border: 1px solid ${Colors.edward};
  margin-top: 20px;
  width: 45vw;
  display: flex;
  flex-direction: column;
`
export const TextChatHeader = styled.div`
  background: ${Colors.concrete};
  border-bottom: 1px solid ${Colors.edward};
  display: flex;
  font-weight: bold;
  align-items: center;
  justify-content: space-between;
  flex: 1;
  padding: 0 25px;
`

export const Header = styled.div`
  font-size: 18px;
`

export const TextChatFooter = styled.div`
  background: ${Colors.concrete};
  border-top: 1px solid ${Colors.edward};
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 15px;
  flex: 1;
`

export const TextChatBody = styled.div`
  flex: 12;
  overflow: auto;
`

export const AvatarContainer = styled.div`
  display: flex;
  & > div {
    margin: 0 1px;
  }
`

export const TextArea = styled(Textarea)`
  background: ${Colors.white};
  border-radius: 100px;
  border: 1px solid ${Colors.edward};
  font-size: 13px;
  padding: 10px 20px;
  resize: none;
  width: 100%;
  outline: none;
`

export const DayInformation = styled.div`
  display: flex;
  justify-content: center;
  font-weight: bold;
  color: ${Colors.ecstasy};
  margin-top: 5px;
`
