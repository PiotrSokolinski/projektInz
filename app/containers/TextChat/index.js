/**
 *
 * TextChat
 *
 */

import React, { useState, useRef } from 'react'
import PropTypes from 'prop-types'
import map from 'lodash/map'
import { FormattedMessage, injectIntl } from 'react-intl'

import Message from 'components/Message'
import UserAvatar from 'components/UserAvatar'
import messages from './messages'
import * as Styled from './styled'

const members = [
  {
    id: 1,
    avatarUrl: '',
  },
  {
    id: 2,
    avatarUrl: '',
  },
  {
    id: 3,
    avatarUrl: '',
  },
  {
    id: 4,
    avatarUrl: '',
  },
]

const TextChat = ({ intl }) => {
  const [newMessage, setNewMessage] = useState('')
  const textArea = useRef(null)
  const setNewMessageText = event => {
    setNewMessage(event.target.value)
  }
  return (
    <Styled.Container>
      <Styled.TextChatHeader>
        <Styled.Header>
          <FormattedMessage {...messages.header} />
        </Styled.Header>
        <Styled.AvatarContainer>
          {map(members, (member, index) => (
            <UserAvatar size="small" key={`el-avatar-${member.id}-${index}`} image={member.avatarUrl} />
          ))}
        </Styled.AvatarContainer>
      </Styled.TextChatHeader>
      <Styled.TextChatBody>
        <Styled.DayInformation>Tusday, 28.10.2019</Styled.DayInformation>
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner={false} time="17:10" />
        <Message
          text="MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage"
          isOwner={false}
          time="17:10"
        />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner={false} time="17:10" />
        <Message
          text="MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage"
          isOwner={false}
          time="17:10"
        />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner={false} time="17:10" />
        <Message
          text="MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage"
          isOwner={false}
          time="17:10"
        />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner={false} time="17:10" />
        <Message
          text="MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage"
          isOwner={false}
          time="17:10"
        />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner={false} time="17:10" />
        <Message
          text="MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage"
          isOwner={false}
          time="17:10"
        />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner time="17:10" />
        <Message text="Message" isOwner={false} time="17:10" />
        <Message
          text="MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage"
          isOwner={false}
          time="17:10"
        />
        <Message text="Message" isOwner time="17:10" />
      </Styled.TextChatBody>
      <Styled.TextChatFooter>
        <Styled.TextArea
          inputRef={textArea}
          maxRows={3}
          onInput={setNewMessageText}
          placeholder={intl.formatMessage(messages.textAreaPlaceholder)}
          spellCheck="false"
          value={newMessage}
        />
        <Styled.AttachmentIcon size="25" />
        <Styled.SendButton type="submit" inverted stable>
          <FormattedMessage {...messages.send} />
        </Styled.SendButton>
      </Styled.TextChatFooter>
    </Styled.Container>
  )
}

TextChat.propTypes = {
  intl: PropTypes.object,
}

export default injectIntl(TextChat)
