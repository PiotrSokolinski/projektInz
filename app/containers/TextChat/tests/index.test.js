/**
 *
 * Tests for TextChat
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react'
import { render } from '@testing-library/react'
import { IntlProvider } from 'react-intl'

import TextChat from '../index'
import { DEFAULT_LOCALE } from '../../../i18n'

describe('<TextChat />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error')
    render(
      <IntlProvider locale={DEFAULT_LOCALE}>
        <TextChat />
      </IntlProvider>,
    )
    expect(spy).not.toHaveBeenCalled()
  })

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale={DEFAULT_LOCALE}>
        <TextChat />
      </IntlProvider>,
    )
    expect(firstChild).toMatchSnapshot()
  })
})
