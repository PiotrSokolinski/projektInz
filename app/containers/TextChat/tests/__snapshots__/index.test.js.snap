// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextChat /> Should render and match the snapshot 1`] = `
<div
  class="styled__Container-laciss-2 ffZdcs"
>
  <div
    class="styled__TextChatHeader-laciss-3 jyyxCZ"
  >
    <div
      class="styled__Header-laciss-4 dVJEuL"
    >
      <span>
        Chat with your Group
      </span>
    </div>
    <div
      class="styled__AvatarContainer-laciss-7 gWEXMY"
    >
      <div
        class="small styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="small styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="small styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="small styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
    </div>
  </div>
  <div
    class="styled__TextChatBody-laciss-6 kcpCJR"
  >
    <div
      class="styled__DayInformation-laciss-9 fWweLp"
    >
      Tusday, 28.10.2019
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 hiCLbP"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 jALbcv"
      >
        MessageMessage MessageMessa geMess ageMessag eMessa geMessag eMessageMessa geMessage MessageMessageM essageMessageMessag eMessageM essageMessa geMessageMessageMessageMessage Message MessageMe ssageMessageMessageMe ssageMessag eMessage
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
    <div
      class="styled__Container-lr2zll-0 gaFdcr"
    >
      <div
        class="tiny styled__Container-vinm4t-0 gDIJTt"
      >
        <img
          alt="avatar"
          class="avatar-image"
          src="IMAGE_MOCK"
        />
      </div>
      <div
        class="styled__MessageCloud-lr2zll-1 drMwnc"
      >
        Message
      </div>
      <div
        class="styled__MessageTime-lr2zll-2 gTomPT"
      >
        17:10
      </div>
    </div>
  </div>
  <div
    class="styled__TextChatFooter-laciss-5 cnAoYg"
  >
    <textarea
      class="styled__TextArea-laciss-8 exIibG"
      placeholder="Start typing..."
      spellcheck="false"
      style="height: 0px;"
    />
    <svg
      aria-hidden="true"
      class="styled__AttachmentIcon-laciss-0 fRuilz StyledIconBase-sc-bdy9j4 jbPeIR"
      fill="currentColor"
      focusable="false"
      height="25"
      viewBox="0 0 16 16"
      width="25"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M10.404 5.11L9.389 4.096 4.314 9.17a2.152 2.152 0 103.045 3.044l6.09-6.089a3.588 3.588 0 00-5.075-5.074L1.98 7.444l-.014.013c-1.955 1.955-1.955 5.123 0 7.077s5.123 1.954 7.078 0l.013-.014.001.001 4.365-4.364-1.015-1.014-4.365 4.363-.013.013c-1.392 1.392-3.656 1.392-5.048 0a3.572 3.572 0 01.014-5.06l-.001-.001L9.39 2.065c.839-.84 2.205-.84 3.045 0s.839 2.205 0 3.044l-6.09 6.089a.718.718 0 01-1.015-1.014l5.075-5.075z"
      />
    </svg>
    <button
      class="styled__SendButton-laciss-1 iPFTwN styled__StyledButton-sc-1gb27vo-0 rmQmn"
      type="submit"
    >
      <span>
        Send
      </span>
    </button>
  </div>
</div>
`;
